extensions/amp-story/1.0/amp-story-access.js

```diff
@@ -136,6 +146,20 @@ export class AmpStoryAccess extends AMP.BaseElement {
         .measureMutateElement(this.element, measurer, mutator);
   }
 
+  /**
+   * Handles click events and maybe closes the paywall.
+   * @param {!Event} event
+   * @private
+   */
+  onClick_(event) {
+    const el = dev().assertElement(event.target);
+
+    // Closes the menu if click happened outside of the main container.
+    if (!closest(el, el => el === this.contentEl_, this.element)) {
+      this.storeService_.dispatch(Action.TOGGLE_ACCESS, false);
```

Comment:
What happens if the user clicks outside the amp-access container after subscribing to the paywall (in the confirmation page)? Will it still set the access to `false`?

Reply:
The listener is set on the element (paywall itself) which is hidden whenever `ACCESS_STATE` is `false`, so a user should not be able to click on it.